### DISCUSSION TITLE
ExplorerPatcher not working on Windows 10 after the latest release

### DISCUSSION THREAD

**Discussion by Tiagoquix** [valinet/ExplorerPatcher#2124] [General]
[Release 22621.2283.57.1](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2283.57.1_a7b29c5) makes the File Explorer crash on Windows 10 22H2 19045.3448 (OS Build 10.0.19045.3448).

The [previous release](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.1992.56.3_5bda71f) works fine.

**Comment by Poopooracoocoo** [comment]
Does EP version 22621.2283.57.2 fix it? Amr Satrio pushed a fix: https://github.com/valinet/ExplorerPatcher/commit/dfee1aee3716efdd7aa1b05f18b1693ec1ce3ce3

**Reply by Tiagoquix** [reply]
Yes; it does. Thank you for letting me know.

**Reply by pyrates999** [reply]
Please close this then.
